### ISSUE TITLE
[accessibility] Use `<button>` or `<a role="button">`

### ISSUE THREAD

**Issue by Malvoz** [stefanocudini/leaflet-gps#23]
`<a class="gps-button">`  button is announced as a "link" using a screen reader (missing [`role="button"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role), [see Leaflet controls](https://github.com/Leaflet/Leaflet/blob/bdd7ebf836201550f6d173943b728098b83f40d5/src/control/Control.Zoom.js#L89-L91)).

[Another option is to use `<button type="button">`](https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/button/button.html), but requires additional styling to reset browser defaults.

---

Other potentially helpful resources:
- [Leaflet Guideline: Test your maps](https://leafletjs.com/examples/accessibility/#test-your-maps)
- [Leaflet Plugin Authoring Guide: Content Accessibility](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md#content-accessibility)

**Comment by stefanocudini** [comment]
great can you send us a little PR about this? I think is oneline around button constructor code.
